commit 4320fe3331f5fb71f2138342224b95d3fd7caf6b (refs/changes/39/3939/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-05-17T14:30:41-07:00 (1 year, 5 months ago)
    
    MANTA-3684 manta-scripts needs to work around default logadm.conf in new base image
    Reviewed by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
    Approved by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>

diff --git a/deps/manta-scripts b/deps/manta-scripts
index dc593cd..cd8d6f1 160000
--- a/deps/manta-scripts
+++ b/deps/manta-scripts
@@ -1 +1 @@
-Subproject commit dc593cd90475e391c6fce874130f5d522dc0cc54
+Subproject commit cd8d6f194ba34dfaa4b0740dfbd11147041dd13e
